lib/plugins/aws/package/compile/events/cloudWatchLog.js

```diff
@@ -54,14 +55,22 @@ class AwsCompileCloudWatchLogEvents {
               FilterPattern = '';
             }
 
-            if (logGroupNames.includes(LogGroupName)) {
+            cloudWatchLogEventNumberMap[LogGroupName] =
+              cloudWatchLogEventNumberMap[LogGroupName] || 0;
+            cloudWatchLogEventNumberMap[LogGroupName]++;
+            if (
+              cloudWatchLogEventNumberMap[LogGroupName] >
+              CLOUDWATCHLOG_LOG_GROUP_EVENT_PER_FUNCTION_LIMIT
+            ) {
               const errorMessage = [
-                `"${LogGroupName}" logGroup for cloudwatchLog event is duplicated.`,
-                ' This property can only be set once per CloudFormation stack.',
+                `Only ${CLOUDWATCHLOG_LOG_GROUP_EVENT_PER_FUNCTION_LIMIT} subscription filter can be configured per log group.`,
+                ` For "${LogGroupName}" you're attempting to configure over limit.`,
```

Comment:
Let's rephrase this line as

` You're attempting to configure more subscription filters than allowed for "${LogGroupName}".`